### PR TITLE
white list location

### PR DESCRIPTION
**Pull request by Dengjianping**
This is not going to be merged. Just show the diff content.
If everything looks good, I will submit a pr to orml.

### CODE REVIEW

**Comment by stechu** [comment]
Looks good to me.

File: xtokens/src/lib.rs [hunk 1]
```diff
@@ -88,6 +94,9 @@ pub mod module {
 		/// XCM executor.
 		type XcmExecutor: ExecuteXcm<Self::Call>;
 
+		///
```
Comment (ghzlatarev) [inline xtokens/src/lib.rs]: You're missing the comment here.

**Comment by stechu** [comment]
what is the status of this PR?

**Comment by Dengjianping** [comment]
> what is the status of this PR?

It has been merged into xtoken.
I'm going close it.

**Pull request closed without merging**